### ISSUE TITLE
Can't access internet within dlite container

### ISSUE THREAD

**Issue by mccall114** [nlf/dlite#222]
## Bug Reports

- dlite version in use (run `dlite --version`): dlite 1.1.5, installed on Mac OS X El Capitan 10.11.6

- expected behavior: within docker, expecting `curl google.com` to return results. also getting connection refused when running `docker pull` or `docker import`

- actual behavior: `curl.google.com` hangs then can't resolve host. same for directly curling the IP

- steps to reproduce: `docker run --rm -it <container> bash` then trying to curl. Docker image is a modified Ubuntu 14.04.4 LTS

This is my /etc/resolv.conf within the container:
nameserver 8.8.8.8
nameserver 192.168.64.1 # eth0

If I run this with regular Docker (aka not dlite), it works fine. It also works fine on a 2nd machine that has both dlite and regular Docker installed. Is there some dlite config that somehow got modified or is missing? Has anyone else experienced this?

**Comment by nlf** [comment]
to document this here (@mccall114 and i have been working on this via email) we haven't been able to determine exactly what's going on.

host to vm and vm to host communication works correctly, vm to internet fails. the vm's routing table is correct and the host's appears to be.

**Comment by mccall114** [comment]
@nlf, is dlite using xhyve at this stage in 1.1.5? Is there any chance that upgrading to a later version of dlite would fix this (i.e. use hyperkit instead)?

**Comment by nlf** [comment]
yup, it sure is, and you're right trying out the 2.0 beta is probably your best bet. it already uses hyperkit in the latest betas.

you'd have to completely remove dlite 1.1.5, then download the latest beta and put it on your path. `brew install libev` then run `dlite init` and `dlite start` once that finishes. with some luck that'll have you up and running.
